test/relayclient/KnownRelaysManager.test.ts

```diff
@@ -168,21 +182,29 @@ contract('KnownRelaysManager 2', function (accounts) {
       await stakeManager.unauthorizeHub(accounts[4], relayHub.address)
     })
 
+    after(async function () {
+      await stopRelay(relayProcess)
+    })
+
     it('should consider all relay managers with stake and authorization as active', async function () {
       await knownRelaysManager.refresh()
-      const relays = knownRelaysManager.getRelays()
-      assert.equal(relays.length, 2)
-      assert.equal(relays[0].relayUrl, 'stakeAndAuthorization1')
-      assert.equal(relays[1].relayUrl, 'stakeAndAuthorization2')
+      const preferredRelays = knownRelaysManager.knownRelays[0]
+      const activeRelays = knownRelaysManager.knownRelays[1]
+      assert.equal(preferredRelays.length, 1)
```

Comment:
nitpick: a single `assert.deepEqual` can compare the entire structure

Reply:
This is about expicitness, not shortness